builder_data.yaml

```diff
@@ -22,3 +24,4 @@ yum_repo_command: |
 osx_signing_cert: "Developer ID Installer: PUPPET LABS, INC. (VKGLGN2B6Y)"
 osx_signing_keychain: "/Users/jenkins/Library/Keychains/signing.keychain"
 osx_signing_server: 'osx-signer.delivery.puppetlabs.net'
+ips_signing_server: 'rama.delivery.puppetlabs.net'
```

Comment:
there's inconsistent quoting in `saturn.delivery.puppetlabs.net` vs. `'rama.delivery.puppetlabs.net'`. Not sure what's better, but we should be consistent.